### PR TITLE
fixes #3

### LINKED ISSUE #3: Primes must be positive
As seen in class

### PR DESCRIPTION
**Pull request by jchiang87**
Hey @drphilmarshall,  I've just pushed these fixes can you code review?

### CODE REVIEW

**Comment by drphilmarshall** [comment]
Say please.

**Comment by jchiang87** [comment]
<Pleeaase?>

File: tests/test_Primes.py [hunk 1]
```diff
@@ -24,5 +24,17 @@ def test_five_is_prime(self):
         "Test that five is prime."
         self.assertTrue(is_prime(5))
 
+    def test_one_is_not_prime(self):
```
Comment (drphilmarshall) [inline tests/test_Primes.py]: Should we refactor the following three tests into one?

Comment (jchiang87) [inline tests/test_Primes.py]: we could do that, but I think it's fine as is.  We can refactor these tests later.

**Pull request merged**